prow/cluster/jobs/istio-private/istio/istio-private.istio.master.gen.yaml

```diff
@@ -1196,8 +1202,11 @@ postsubmits:
         - entrypoint
         - prow/integ-suite-kind.sh
         - --node-image
-        - kindest/node:v1.16.3
+        - kindest/node:v1.16.4
         - test.integration.kube.presubmit
+        env:
+        - name: INTEGRATION_TEST_FLAGS
+          value: ' --istio.test.retries=1 '
```

Comment:
Minor nit. It seems you use a single quote in some areas, and a double quote in others.

Reply:
Its auto generated, I have no choice here. this is how the yaml library handles things to properly escape